src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResourceTest.kt

```diff
@@ -1145,12 +1130,7 @@ class ReportResourceTest : SqsIntegrationTestBase() {
             true,
           )
 
-        getDomainEvents(1).let {
-          assertThat(it.map { message -> message.eventType to message.additionalInformation?.source })
-            .containsExactlyInAnyOrder(
-              "incident.report.amended" to InformationSource.DPS,
-            )
-        }
+        assertThatDomainEventWasSent("incident.report.amended", "IR-0000000001124143")
```

Comment:
Too verbose to use the enum maybe?

Reply:
the value inside the event that's sent is a string so the enum would need to be converted to a string… which feels less accurate cos you then never truly test the actual value…?

Reply:
just had a look and the enum variant name is equally long :)

Reply:
Yeah that is long, but you could call `.value` on the enum AFAICT. I'd use the constant but I don't mind using these strings as I imagine they're pretty much stable :)

Reply:
while not terrible, i see the downside of something like this
```kotlin
assertThatDomainEventWasSent(INCIDENT_REPORT_AMENDED, "IR-0000000001124143")
```
as tests not picking up as many typos should they happen

it's for a similar reason that i like that we have a fixed clock so that all json responses can use an exact string representation rather than `$now`